src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1261,8 +1270,7 @@ static void lb_call_on_retry_timer_locked(grpc_exec_ctx* exec_ctx, void* arg,
   glb_policy->retry_timer_active = false;
   if (!glb_policy->shutting_down && error == GRPC_ERROR_NONE) {
     if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
-      gpr_log(GPR_INFO, "Restaring call to LB server (grpclb %p)",
-              (void*)glb_policy);
+      gpr_log(GPR_INFO, "[grpclb %p] Restaring call to LB server", glb_policy);
```

Comment:
typo: "Restaring" instead of "Restarting"

Reply:
Done